### DISCUSSION TITLE
[neovim] copilot is enabled, panel works but doesn't suggest any code after installing astronvim

### DISCUSSION THREAD

**Discussion by oriongonza** [community/community#53165] [Copilot Conversations]
### Select Topic Area

Bug

### Body

This is my config code
```
  {
    "github/copilot.vim",
    event = "BufRead",
  }
```
Copilot status says it's online and enabled, and the panel works as expected. Just no code completion.

Used to work fine until I installed astronvim configuration.

**Comment by Akash1134** [comment]
Hi @dev-ardi ,

Could you please try updating to the latest version of the Copilot extension (v1.83.41) if you haven't so far? Please restart your IDE and re-authenticate if required.

Additionally, please also make sure that you are logged into GitHub.com with the GitHub account that has the active Copilot subscription (this is important if you have more than one GitHub account i.e. one for work and another personal one.)

Also, please remove the Copilot Labs extension if you have it installed. Having both extensions at the same time is also suspected to cause problems, as the built-in GitHub Authentication extension may end up attempting to authenticate one but not the other.

Thanks!

**Reply by vinitpayal** [reply]
Same for me actually and no I haven't been using the Copilot Labs extension on my neovim setup.

**Reply by wybert** [reply]
same here
